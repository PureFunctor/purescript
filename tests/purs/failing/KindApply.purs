-- @shouldFailWith CannotApplyTypeOnType

module Main where

type Fail = Array @Int
