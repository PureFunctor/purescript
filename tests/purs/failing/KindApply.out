Error found:
in module [33mMain[0m
at tests/purs/failing/KindApply.purs:5:13 - 5:23 (line 5, column 13 - line 5, column 23)

  A type of type:
  [33m              [0m
  [33m  Type -> Type[0m
  [33m              [0m
  cannot be applied to the type:
  [33m     [0m
  [33m  Int[0m
  [33m     [0m
  as it is a monotype.

while inferring the kind of [33mArray @Int[0m
in type synonym [33mFail[0m

See https://github.com/purescript/documentation/blob/master/errors/CannotApplyTypeOnType.md for more information,
or to contribute content related to this error.

